### PR TITLE
F OpenNebula/one#6112: remote auth config

### PR DESCRIPTION
**Pull request by jloboescalona2**
this PR applies to:
- master
- one-6.6-maintenance

### CODE REVIEW

File: source/installation_and_configuration/opennebula_services/fireedge.rst [hunk 1]
```diff
@@ -84,6 +84,17 @@ The FireEdge server configuration file can be found in ``/etc/one/fireedge-serve
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
 | ``guacd/host``                            | ``localhost``                  | Connection hostname/IP of guacd server             |
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
+| ``auth``                                  | ``opennebula``                 | Authentication driver for incoming requests:       |
+|                                           |                                | **opennebula** the authentication will be done by  |
+|                                           |                                | the opennebula core using the driver defined for   |
+|                                           |                                | user. **remote** performs the login based on a     |
```
Comment (FrederickBor) [inline source/installation_and_configuration/opennebula_services/fireedge.rst:90]: Missing article

```suggestion
|                                           |                                | the user. **remote** performs the login based on a     |
```

File: source/installation_and_configuration/opennebula_services/fireedge.rst [hunk 2]
```diff
@@ -84,6 +84,17 @@ The FireEdge server configuration file can be found in ``/etc/one/fireedge-serve
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
 | ``guacd/host``                            | ``localhost``                  | Connection hostname/IP of guacd server             |
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
+| ``auth``                                  | ``opennebula``                 | Authentication driver for incoming requests:       |
+|                                           |                                | **opennebula** the authentication will be done by  |
+|                                           |                                | the opennebula core using the driver defined for   |
```
Comment (FrederickBor) [inline source/installation_and_configuration/opennebula_services/fireedge.rst:89]: I believe this should be in Uppercase as it is the company name

```suggestion
|                                           |                                | the OpenNebula core using the driver defined for   |
```

File: source/installation_and_configuration/opennebula_services/fireedge.rst [hunk 3]
```diff
@@ -84,6 +84,17 @@ The FireEdge server configuration file can be found in ``/etc/one/fireedge-serve
...
+|                                           |                                | user. **remote** performs the login based on a     |
+|                                           |                                | Kerberos X-Auth-Username header provided by        |
+|                                           |                                | authentication backend                             |
++-------------------------------------------+--------------------------------+----------------------------------------------------+
+| ``auth_redirect``                         |                                | This configuration is for the login button         |
+|                                           |                                | redirect. The available options are: "**/**",      |
+|                                           |                                | "**.**" or a "**URL**"                             |
```
Comment (FrederickBor) [inline source/installation_and_configuration/opennebula_services/fireedge.rst:96]: URL does not need quotation marks as it is not a real value

```suggestion
|                                           |                                | "**.**" or a **URL**                             |
```

**Review by FrederickBor — CHANGES_REQUESTED** [review]

File: source/installation_and_configuration/opennebula_services/fireedge.rst (hunk 1 above)
Comment (jloboescalona2) [reply source/installation_and_configuration/opennebula_services/fireedge.rst]: done!

File: source/installation_and_configuration/opennebula_services/fireedge.rst (hunk 3 above)
Comment (jloboescalona2) [reply source/installation_and_configuration/opennebula_services/fireedge.rst]: done!

File: source/installation_and_configuration/opennebula_services/fireedge.rst (hunk 2 above)
Comment (jloboescalona2) [reply source/installation_and_configuration/opennebula_services/fireedge.rst:89]: done!

Comment (FrederickBor) [reply source/installation_and_configuration/opennebula_services/fireedge.rst:89]: This is still not done

**Review by FrederickBor — CHANGES_REQUESTED** [review]

**Review by FrederickBor — APPROVED** [review]

**Review by tinova — APPROVED** [review]

**Pull request merged**